src/selectors/prometheus/selectors.js

```diff
@@ -0,0 +1,22 @@
+import { get } from 'lodash';
+
+export const getConsumers = (kind, results, nameLabel, formatLabel) => {
+  if (!results) {
+    return null;
+  }
+  const result = get(results, 'data.result');
+  return result.map(r => ({
+    kind,
+    name: r.metric[nameLabel],
+    usage: r.value[1],
+    label: formatLabel ? formatLabel(r.value[1]) : r.value[1],
+  }));
+};
+
+export const getCapacityStats = response => {
+  const value = get(response, 'data.result[0].value[1]');
+  if (Number.isNaN(parseFloat(value))) {
+    return null;
+  }
+  return Number(value);
```

Comment:
Do we expect whitespaces in the value? Number is more stronger so it could still return `NaN`. We can just use the parseFloat value if we want to be more lenient.

Reply:
Not sure about whitespaces...i would say no and stick with the Number for now

Reply:
I just saw https://stackoverflow.com/questions/12227594/which-is-better-numberx-or-parsefloatx. Each one of them has it's quirks. Can we check for NaN of both `parseFloat` and `Number` ?

maybe it would be worth to also make a generic parsing function out of it. It would return number or null only.

Reply:
I think a number-like value is returned, not a string

Reply:
ok, still better to check against `Number('') === 0`

Reply:
new function in `utils` - `parseNumber` is using combination of `parseFloat`/`Number`/`Number.isNaN` . WDYT ?

Reply:
+1